llama-index-integrations/vector_stores/llama-index-vector-stores-lancedb/llama_index/vector_stores/lancedb/base.py

```diff
@@ -232,17 +328,21 @@ def query(
             lance_query.refine_factor(self.refine_factor)
 
         results = lance_query.to_pandas()
+
+        if len(results) == 0:
+            raise Warning("query results are empty..")
+
         nodes = []
         for _, item in results.iterrows():
             try:
                 node = metadata_dict_to_node(item.metadata)
-                node.embedding = list(item[self.vector_column_name])
+                node.embedding = None
```

Comment:
why do we remove the embedding?

Reply:
The other providers were not returning the embedding either, so for consistency purposes. I can add it back tho

Reply:
Yea kind of just not wanting to break anyone relying on it